doc/specs/#976 - VT52 escape sequences.md

```diff
@@ -0,0 +1,141 @@
+
+Changing back from VT52 mode to ANSI mode would need to be achieved with a separate VT52 command, since the VT100 CSI mode sequences would no longer be active. This would be handled in the same place as the other VT52 commands, in the `OutputStateMachineEngine`, and then passed on to the mode selection method in the `AdaptDispatch` class described above (essentially the equivalent of the DECANM private mode being set).
+
+### Additional VT52 Commands
+
+Most of the missing VT52 functionality can be implemented in terms of existing VT100 methods.
+ 
+* The _Cursor Up_, _Cursor Down_, _Cursor Left_, and _Cursor Right_ commands are already implemented.
+* The _Enter Graphics Mode_ and _Exit Graphics Mode_ commands can probably use the existing `DesignateCharset` method, although this would require a new `VTCharacterSets` option with a corresponding table of characters (see below).
+* The _Reverse Line Feed_ command can use the existing `ReverseLineFeed` method.
+* The _Erase to End of Display_ and _Erase to End of Line_ commands can use the existing `EraseInDisplay` and `EraseInLine` methods.
+* The _Cursor Home_ and _Direct Cursor Address_ commands can probably be implemented using the `CursorPosition` method. Technically the _Direct Cursor Address_ has different rules for the boundary conditions, but nobody seems to get that right, so it's probably not that big a deal.
+* The _Identify_ command may be the only one that doesn't build on existing functionality, but it should be a fairly trivial addition to the `AdaptDispatch` class.
+* The _Enter Keypad Mode_ and _Exit Keypad Mode_ commands can use the existing `SetKeypadMode` method, assuming the `Terminal Input` class already knows to generate different sequences when in VT52 mode.
+* The _Enter ANSI Mode_ command can just call through to the new mode selection method in the `AdaptDispatch` class as discussed in the _Changing Modes_ section above.
+
+There are also a few VT52 print commands, but those are not technically part of the core command set, so they can probably be considered out of scope for now.
+
+### Graphic Mode Character Set
+
+The table below lists suggested mappings for the _Graphics Mode_ character set, based on the descriptions in the [VT102 User Guide](https://vt100.net/docs/vt102-ug/table5-15.html).
+
+Note that there is only the one _fraction numerator_ character in Unicode, so superscript digits have instead been used for the numerators 3, 5, and 7. There are also not enough _horizontal scan line_ characters (for the _bar at scan x_ characters), so each of them is used twice to cover the full range.

```

Comment:
I momentarily thought we should do something special to signal the renderer to draw a single pixel tall line at the various percentages... but then I thought that's probably too much effort and this is a decent enough solution for now.

Reply:
That would be really cool, but yeah, probably not a priority at this point.